README.md

```diff
@@ -1,4 +1,4 @@
-# prefect-project-deploy (WORK IN PROGRESS)
+# prefect-project-deploy
```

Comment:
i think we should use a consistent naming convention for our custom modules - in this case a custom GHA

do we need `prefect-`? also for [this guy](https://github.com/PrefectHQ/gha-retrieve-alembic-revisions), we did `gha-<name of your action>`

Reply:
what about `prefecthq/actions-deploy-project`? we could update the other one to follow this convention

i think other GHA authors set up repos in an action-specific org, so it's something like:

```
actions/checkout@v2
google-github-actions/setup-gcloud@v1
```

but we dont need to do that if we just add the `actions` prepend?  

also i always think of GHA as just a function, so might be worth making the action name a verb, like "prefecthq/actions-do-something"

these are all suggestions by the way, nothing required